contrib/demo-rgb.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# There might be a need to download build-essentials, cmake, python3 and python3-dev
+sudo apt install build-essentials
+sudo apt install cmake
+sudo apt install python3
+sudo apt install python3-dev
```

Comment:
We need `-y` flag to prevent requiring user input

```suggestion
sudo apt install -y build-essentials cmake python3 python3-dev
```